public/locales/en/translation.json

```diff
@@ -316,6 +316,13 @@
     },
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
+    },
+    "contactUsInfo": {
+      "header": "Leave us a message",
+      "description": "Torquatos nostros? quos tu tam crudelis fuisse, nihil oportere nimium nos causae confidere, sed animo etiam erga nos causae confidere, sed uti oratione perpetua malo quam ostendis sed quia dolor repellendus temporibus autem.",
+      "location": "9094 Bay Meadows Street Conyers, GA 30012",
+      "phone": "+229-955-5388-336",
```

Comment:
im wondering this again, if you delete this from the AR and TR translations, does it fallback to the english one? it should i believe, then you dont have to copy-paste redundant info

Reply:
it works I think when I remove these lines from Ar, it is switching the website to Arabic. But navbar is making everything fall into English Jamil will check that issue.

Reply:
can you get a TR translation from someone in this PR too.

Reply:
no TR for us :D

Reply:
Anadili Türkçe olan birisi yok.